model/parsley/parsley.go

```diff
@@ -0,0 +1,43 @@
+package parsley
```

Comment:
I'm typically hesitant to have new packages introduced but i see why this is useful, I think it's separate enough that it won't cause issues